fdt/items.py

```diff
@@ -41,11 +42,10 @@ def new_property(name: str, raw_value: bytes) -> object:
     elif len(raw_value) and len(raw_value) % 4 == 0:
         obj = PropWords(name)
         # Extract words from raw value
-        for i in range(0, len(raw_value), 4):
-            obj.append(unpack(">I", raw_value[i:i + 4])[0])
+        obj.data = [BIGENDIAN_WORD.unpack(raw_value[i:i + 4])[0] for i in range(0, len(raw_value), 4)]
```

Comment:
There are two changes here.  Firstly, use `BIGENDIAN_WORD.unpack` so that we don't have to compile the format a few million times.  If we use `Struct` at all, this cannot be controversial (it speeds up reading big files by x2).

Secondly, rather than use `.append` on what we know is an initially empty list, construct the list with a single comprehension and then assign it.  It could be argued that this is breaking the encapsulation of PropWords - if so, I can write a mutator for PropWords which takes the list and assigns it (it's worth another x2 on performance).